tests/rsa_tests.rs

```diff
@@ -142,24 +142,19 @@ fn test_signature_rsa_pkcs1_sign_output_buffer_len() {
         include_bytes!("../src/rsa/signature_rsa_example_private_key.der");
     let key_pair = rsa::KeyPair::from_der(PRIVATE_KEY_DER).unwrap();
 
-    // The output buffer is one byte too short.
-    let mut signature = vec![0; key_pair.public().modulus_len() - 1];
-
-    assert!(key_pair
-        .sign(&signature::RSA_PKCS1_SHA256, &rng, MESSAGE, &mut signature)
-        .is_err());
-
-    // The output buffer is the right length.
-    signature.push(0);
-    assert!(key_pair
-        .sign(&signature::RSA_PKCS1_SHA256, &rng, MESSAGE, &mut signature)
-        .is_ok());
-
-    // The output buffer is one byte too long.
-    signature.push(0);
-    assert!(key_pair
-        .sign(&signature::RSA_PKCS1_SHA256, &rng, MESSAGE, &mut signature)
-        .is_err());
+    // Wen the output buffer is not exactly the right length, `sign()` returns
```

Comment:
Typo: 
```suggestion
    // When the output buffer is not exactly the right length, `sign()` returns
```

Reply:
Thanks. I fixed it and pushed a new version of this set of commits.